adage/analyze/management/commands/import_gene_sample_expr.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+"""
+This management command reads an input file of gene-sample expression values
+and loads the valid data into database.  It should be invoked like this:
```

Comment:
Quick minor thing: I think this sentence would flow better if it was either "... and loads the valid data into **the** database" or "... and loads valid data into database." I kind of prefer this first one, though.